### ISSUE TITLE
accessible name calculation does not correctly handle empty aria-labelledby

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-core#372]
As described here

https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te

An empty aria-labelledby attribute should be ignored, yet this code

`<button aria-labelledby="">Button</button>`

is raising a violation
